src/docs/product/sentry-basics/search/index.mdx

```diff
@@ -10,6 +10,16 @@ description: "Learn more about Search, which is available in several Sentry feat
 
 Search is available on several features throughout [sentry.io](https://sentry.io), such as **Issues**, **Discover** and **Dashboards**.
 
+## Filters
+
+Page filters enable you to narrow down the results shown on a page by restricting data to a certain project, environment, or date range.
+
+  ![Page Filters](search-page-filters.png)
+
+These filters are normally only persistent within the current page, but you can choose to "lock" the current filter selection such that any changes made to the "locked" filter will be persisted throughout the application and even remembered between sessions.
```

Comment:
This sentence is pretty long so I've broken it up. I also added a couple words to speak to current and change behaviour.

```suggestion
Typically, these filters only persist within the current page, but you can choose to "lock" the current filter selection. When you do this, the locked filter and any changes to it are persisted throughout the application and remembered between sessions. 
```